### PR TITLE
Update example files with comments

### PR DESCRIPTION
**Pull request by znicholls**
After discussions with @durack1, I have put some comments into the examples which I think would help someone who has no experience with CMOR get started.

### CODE REVIEW

**Comment by durack1** [comment]
@mauzey1 @doutriaux1 some comments have been added to these files to make them easier to use, if the checks pass (they should no code is changed) can this be merged?

File: Test/test_doc.py [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
-cmor.setup(inpath='Tables', netcdf_file_action=cmor.CMOR_REPLACE_4)
+cmor.setup(
+	inpath='./cmip6-cmor-tables/Tables',  # this has to point to the CMIP6 CMOR tables path
+	netcdf_file_action=cmor.CMOR_REPLACE_4
+)
 
-cmor.dataset_json("Test/CMOR_input_example.json")
-
-table = 'CMIP6_Amon.json'
+cmor.dataset_json("common_user_input.json")
```
[the remark this answers is not in the document]
Comment (znicholls) [reply Test/test_doc.py]: Happy to change but I don't understand why there is a renaming being done?

File: Test/test_doc.py [hunk 2]
```diff
@@ -1,29 +1,41 @@
 import cmor
 
-cmor.setup(inpath='Tables', netcdf_file_action=cmor.CMOR_REPLACE_4)
+cmor.setup(
+	inpath='./cmip6-cmor-tables/Tables',  # this has to point to the CMIP6 CMOR tables path
```
[the remark this answers is not in the document]
Comment (znicholls) [reply Test/test_doc.py]: I thought that some users may want to git their CMOR input scripts/code/notes etc. Putting another git repo (i.e. the CMIP6 tables) within their git repo can get very confusing so I wanted to show that you can put the tables anywhere on disk and that you just need `inpath` to point to where those tables are.

[the remark this answers is not in the document]
Comment (mauzey1) [reply Test/test_doc.py]: @doutriaux1 What are your thoughts on this?

[the remark this answers is not in the document]
Comment (doutriaux1) [reply Test/test_doc.py]: I think it is uncessary, especially since the `Tables` directory reflects master of cmip6-cmor-tables repo.

**Comment by znicholls** [comment]
It's definitely unnecessary. The point was to show other users (who would
copy the test_doc.py file into their own set of files) that they don't have
to copy the tables too, they can just change the path to which they point.
Happy to alter if you feel the comment is sufficient explanation.

**Comment by mauzey1** [comment]
@znicholls 
Yes, I think the comment is sufficient for explaining the inpath parameter.  Also, I forgot to add that the input file path should be "Test/CMOR_input_example.json".  That way the test can be run from the cmor directory like all other Python tests.

File: Test/test_doc.py [hunk 3]
```diff
@@ -1,29 +1,41 @@
...
+	netcdf_file_action=cmor.CMOR_REPLACE_4
+)
 
-cmor.dataset_json("Test/CMOR_input_example.json")
-
-table = 'CMIP6_Amon.json'
+cmor.dataset_json("CMOR_input_example.json")
```
Comment (mauzey1) [inline Test/test_doc.py]:
```suggestion
cmor.dataset_json("Test/CMOR_input_example.json")    
```

**Pull request merged**